carRental/src/main/java/com/amigoscode/chohort2/carRental/carProviderCoupon/VM/CarProviderCouponVM.java

```diff
@@ -0,0 +1,28 @@
+package com.amigoscode.chohort2.carRental.carProviderCoupon.VM;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import jakarta.validation.constraints.NotEmpty;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+import java.time.LocalDate;
+
+@Setter
+@Getter
+@Accessors(chain = true)
+public class CarProviderCouponVM {
+    @NotEmpty
```

Comment:
VM classes should implement Serializable;
Accordint to Java Object Serialization Specification it is highly recommended "that all serializable classes other than enum types explicitly declare serialVersionUID values".